src/components/ui/custom/no-users-found.tsx

```
@@ -0,0 +1,86 @@
+                </>
+              ) : (
+                <>
+                  <User2 className="w-5 h-5 mr-2" />
+                  {language === "en" ? "Create New Account" : "Crear Nueva Cuenta"}
+                </>
+              )}
+            </Button>
+            <div className="relative">
+              <div className="absolute inset-0 flex items-center">
+                <div className="w-full border-t border-gray-200"></div>
+              </div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)